.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Test
+
+on: [push, pull_request]
+jobs:
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
+      - name: Copy env
+        run: cp .env.example .env
```

Comment:
Question: the .example file contains a bunch of sample values -- would those be valid during the pipeline execution?

Reply:
Yes.  The CI job passed using the .example.env 

https://github.com/Incredihire/Survey/actions/runs/10853771214

Reply:
Additionally, this a good way to regression test `.env.example`.  This file has to be stable so that any new member of the organization can use it to quickly build the product and develop on it.  If CI fails because something is wrong with `.env.example`, that's an important thing to know

Reply:
Yeah, I think you're right.